### DISCUSSION TITLE
Where does SB fit in my app?

### DISCUSSION THREAD

**Discussion by gbruins-notours** [supabase/supabase#446] [Questions] [answered]
I just discovered SupaBase.  I have an app with a Vue frontend using REST APIs to connect to a Node.js back end (Hapi server), using a Postgres DB (along with Bookshelf.js ORM & knex.js).    Is SB suitable as a 'drop-in' replacement of my current Postgres DB, whereby I can integrate more of it's features over time (like building some real-time data reports, or something like that).   Or is that not the SB sweet spot?

SB compares itself to Firebase, and Firebase has this (old) article called "[Where does Firebase fit in your app?](https://firebase.googleblog.com/2013/03/where-does-firebase-fit-in-your-app.html)".   None of the 3 scenarios outlined in that article describes Firebase as a persistence layer for a server only... they show the client connecting directly to the DB.  This lead me to think SB was similar, and perhaps not a good drop-in DB replacement in my standard client/server/REST API scenario.

**Comment by awalias** [comment, marked as answer]
Hey @gbruins-notours! Thanks for the question

Supabase uses a plain postgres database behind the scenes so you can use Supabase just for the datastore, we give you direct access to the database as well so you can use it however you wish. 

The real power of Supabase is in the autogenerated REST APIs (generated by introspecting your schema), the Auth APIs, and the Realtime API which you can use to subscribe to changes in your DB without polling.

It's a nice idea to have something like the Firebase doc you mentioned, I will def suggest this to the team 👍

**Reply by gbruins-notours** [reply]
That is the answer I was hoping for!    I will swap my 'vanilla' postgres DB with SB and see what happens.

Please take this next comment with a grain of salt because by no means am I a marketing professional, but it seems to me what you are building is postgres with super powers, and positioning yourself as a firebase-like variant actually undersells your offering.   The true market seems to be postgres-as-a-service AND firebase.
